### ISSUE TITLE
Example of atpy where method in Guide for Migrating to Astropy

### ISSUE THREAD

**Issue by mwcraig** [atpy/atpy#68]
It was obvious how to do this once I figured it out but it would have been handy in the migration guide. Something along the lines of this, which also points out that columns are no longer created as attributes of the table:

The `where` method in atpy is replaced by using a boolean array as index to the table. 

``` python
    # atpy
    t_new = t.where((t.id > 10) & (t.ra < 45.4) & (t.flag == 'ok'))

    # astropy.table
    t_new = t[ (t['id'] > 10) & (t['ra'] < 45.4) & (t['flag'] == 'ok') ]
```

**Comment by astrofrog** [comment]
Thanks for spotting this! Feel free to submit pull requests to fix this (and #69) if you have time over the next few days - otherwise I'll do it in a week or so.

**Comment by mwcraig** [comment]
Will have it done later today.
